test/addons_test.go

```diff
@@ -396,6 +398,7 @@ master:
       cpu: 100m
       memory: 256Mi
 data:
+  replicas: 1
   persistence:
     size: 4Gi
   heapSize: 1024m
```

Comment:
Are we simply reducing replicas to reduce time spent? Are we theoretically missing out on any important differences that could occur in changes to our addons by doing so?

Reply:
Time and resources.
A data node basically will hold data for that specific moment we run the test and as we currently don't check replicas or sharding spreaded over multiple data nodes we can live with 1 from my point of view.